sdk/tests/conformance/extensions/webgl-compressed-texture-es3-0.html

```diff
@@ -0,0 +1,146 @@
+** copy of this software and/or associated documentation files (the
+** "Materials"), to deal in the Materials without restriction, including
+** without limitation the rights to use, copy, modify, merge, publish,
+** distribute, sublicense, and/or sell copies of the Materials, and to
+** permit persons to whom the Materials are furnished to do so, subject to
+** the following conditions:
+**
+** The above copyright notice and this permission notice shall be included
+** in all copies or substantial portions of the Materials.
+**
+** THE MATERIALS ARE PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+** EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+** MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+** IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+** CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+** TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+** MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS.
+*/
+
+-->
+
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>WebGL WEBGL_compressed_texture_es3_0 Tests</title>
+<LINK rel="stylesheet" href="../../resources/js-test-style.css"/>
+<script src="../../js/js-test-pre.js"></script>
+<script src="../../js/webgl-test-utils.js"></script>
+</head>
+<body>
+<div id="description"></div>
+<div id="console"></div>
+<script>
+"use strict";
+description("This test ensures WebGL implementations correctly implements WEBGL_compressed_texture_es3_0");
+
+debug("");
+var COMPRESSED_RGB_S3TC_DXT1_EXT              = 0x83F0;
+var COMPRESSED_RGBA_S3TC_DXT1_EXT             = 0x83F1;
+var COMPRESSED_RGBA_S3TC_DXT3_EXT             = 0x83F2;
+var COMPRESSED_RGBA_S3TC_DXT5_EXT             = 0x83F3;
+var COMPRESSED_RGB_PVRTC_4BPPV1_IMG           = 0x8C00;
+var COMPRESSED_RGBA_PVRTC_4BPPV1_IMG          = 0x8C02;
+var ETC1_RGB8_OES                             = 0x8D64;
+var COMPRESSED_R11_EAC                        = 0x9270;
+var COMPRESSED_SIGNED_R11_EAC                 = 0x9271;
+var COMPRESSED_RG11_EAC                       = 0x9272;
+var COMPRESSED_SIGNED_RG11_EAC                = 0x9273;
+var COMPRESSED_RGB8_ETC2                      = 0x9274;
+var COMPRESSED_SRGB8_ETC2                     = 0x9275;
+var COMPRESSED_RGB8_PUNCHTHROUGH_ALPHA1_ETC2  = 0x9276;
+var COMPRESSED_SRGB8_PUNCHTHROUGH_ALPHA1_ETC2 = 0x9277;
+var COMPRESSED_RGBA8_ETC2_EAC                 = 0x9278;
+var COMPRESSED_SRGB8_ALPHA8_ETC2_EAC          = 0x9279;
+
+var wtu = WebGLTestUtils;
+var gl = wtu.create3DContext(undefined, undefined);
+var WEBGL_compressed_texture_es3_0;
+
+var formats = null;
+
+function runTest() {
+  if (!gl) {
+    testFailed("context does not exist");
+  } else {
+    testPassed("context exists");
+
+    var tex = gl.createTexture();
+    gl.bindTexture(gl.TEXTURE_2D, tex);
+
+    var name = wtu.getSupportedExtensionWithKnownPrefixes(gl, "WEBGL_compressed_texture_es3_0");
```

Comment:
Since this is a new extension it shouldn't have a browser prefix. Also, since "name" isn't referenced in the getExtension call below, it wouldn't work even if it did. How about just:

var hasExtension = gl.getSupportedExtensions().indexOf("WEBGL_compressed_texture_es3_0") >= 0;

?